build/pipelines/daily-loc-submission.yml

```diff
@@ -51,13 +63,46 @@ steps:
     outputDirectoryRoot: LocOutput
     appendRelativeDir: true
     pseudoSetting: Included
+    localizationTarget: true
 
-# Saving one of these makes it really easy to inspect the loc output...
-- powershell: 'tar czf LocOutput.tar.gz LocOutput'
-  displayName: 'Archive Loc Output for Submission'
+- pwsh: |-
+    Remove-Item -EA:Ignore -R -Force LocOutput\Terminal.Internal
+    $Files = Get-ChildItem LocOutput -R -Include 'ContextMenu.resw','Resources.resw' | ? FullName -Like '*en-US\*\*.resw'
+    $Files | % { Move-Item -Verbose $_.Directory $_.Directory.Parent.Parent -EA:Ignore }
+    Get-ChildItem LocOutput -R -Filter '*.resw' | % {
+      ((Get-Content $_.FullName) -join "`r`n") + "`r`n" | Set-Content -NoNewline $_.FullName
+    }
+    & tar.exe -c -f LocOutputMunged.tar -C LocOutput .
+    & tar.exe -x -v -f LocOutputMunged.tar
+    rm LocOutputMunged.tar
+    rm -r -fo LocOutput
+    & ./build/scripts/Copy-ContextMenuResourcesToCascadiaPackage.ps1
+  displayName: Move Loc files to the right places
 
-- task: PublishBuildArtifacts@1
-  displayName: 'Publish Artifact: LocOutput'
-  inputs:
-    PathtoPublish: LocOutput.tar.gz
-    ArtifactName: LocOutput
+- pwsh: |-
+    git add **/*.resw
+    git reset HEAD **/en-US/*.resw # Don't check in automated changes to en-US files; too noisy
+    git status
+    git diff --quiet --cached --exit-code
+    If ($LASTEXITCODE -Ne 0) {
+      $Now = Get-Date
+      git commit -m "Localization Updates - $Now"
+      git push origin HEAD:refs/heads/${{parameters.targetBranch}} -f
+      Write-Host "##vso[task.setvariable variable=ChangesPushedToRepo]1"
+    } Else {
+      Write-Host "##vso[task.setvariable variable=ChangesPushedToRepo]0"
+    }
+  displayName: git commit and push
+
+- pwsh: |-
+    Import-Module PSGitHub
+    $PSDefaultParameterValues['*GitHub*:Owner'] = "microsoft"
+    $PSDefaultParameterValues['*GitHub*:RepositoryName'] = "terminal"
+    $PSDefaultParameterValues['*GitHub*:Token'] = ("$(GithubPullRequestToken)" | ConvertTo-SecureString -AsPlainText -Force)
+    $existingPr = Get-GitHubPullRequest -HeadBranch "${{parameters.targetBranch}}"
+    If ($null -Eq $existingPr) {
+      $Now = Get-Date
+      New-GitHubPullRequest -Head "${{parameters.targetBranch}}" -Title "Localization Updates - $Now"
```

Comment:
This cmdlet supports a base branch, but I couldn't get it to work properly in CI. So, I gave up. It always targets main now.